.github/workflows/code_analysis.yaml

```diff
@@ -33,10 +33,6 @@ jobs:
                         name: 'PHPStan'
                         run: vendor/bin/phpstan analyse --ansi --error-format symplify
 
-                    -
-                        name: 'PHPStan for config'
-                        run: composer phpstan-config
```

Comment:
@TomasVotruba I removed it since now it got notice:

```
"phpstan-config" is not defined.

                                            
  Command "phpstan-config" is not defined.
```

and seems removed at

- https://github.com/rectorphp/rector-src/pull/5430